swim/disseminator.go

```diff
@@ -137,7 +137,7 @@ func (d *disseminator) bumpPiggybackCounters(changes []Change) {
 			continue
 		}
 
-		c.p += 1
+		c.p++
```

Comment:
Yes, it's a lint thing.
https://github.com/golang/lint/blob/master/lint.go#L1231